lib/internal/main/mksnapshot.js

```diff
@@ -167,6 +172,10 @@ function main() {
 
   const serializeMainArgs = [process, requireForUserSnapshot, minimalRunCjs];
 
+  if (isExperimentalSeaWarningNeeded()) {
```

Comment:
I added a branch to print it while the SEA blob is being built - I actually think it makes more sense to not print it when the SEA is run now that we have a build phase, as this being experimental is probably only what the builder should know, but not the users (the users might not even know what Node.js is, thinking about what SEA is for, and a warning could be quite confusing for people who don't even know what Node.js is). I'll keep the warning for normal SEAs since others are already testing it, but leave it for snapshots.